index.html

```diff
@@ -18,12 +18,15 @@
             <nav class="nav">
                 <div class="container d-flex justify-content-between">
                     <a href="#" class="my-logo text-decoration-none text--primary weight--700">My Logo</a>
-                    <ul class="d-flex list-style-type-none nav-link weight--500">
+                    <div class="sideBar d-none" id="sideBar">
+                        <i onclick="openBar()" class="fas fa-close"></i>
+                    </div>
+                    <ul class="list-style-type-none nav-link d-none" id="navLink">
                         <li><a href="#Portfolio">Portfolio</a></li>
                         <li><a href="#About">About</a></li>
                         <li><a href="#Contact">Contact</a></li>
                     </ul>
-                    <i class="fa-solid fa-bars text--primary justify-content-center d-flex"></i>
+                    <i class="fa-solid fa-bars text--primary justify-content-center d-flex" onclick="openBar()"></i>
```

Comment:
Good job implementing the mobile menu for mobile screens so far!  Let's however improve upon the following aspects of our design:
- [ ] Kindly ensure that the hamburger and the close button are in the same position when toggled.
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ behave as buttons when the mouse hovers over them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.